templates/app/app.yml

```diff
@@ -3,7 +3,7 @@
 AWSTemplateFormatVersion: 2010-09-09
 Description: Configure the AWSCloudFormationStackSetAdministrationRole to enable use of AWS CloudFormation StackSets.
 Metadata:
-  TemplateVersion: 'v1.0.1'
+  TemplateVersion: 'v1.0.2'
```

Comment:
Should we remove now the `AppDNSDelegatedAccounts` param if its not used?

Reply:
it is used boss. Otherwise we won't know the previous value for createAppInput.AppDNSDelegatedAccounts